src/main/json/r6OperatorCounters.json

```diff
@@ -1,1528 +1 @@
-{
```

Comment:
I'm having trouble running the project at the moment so still need to figure out what my problem is (having a problem with `npm run createjson` but I will post here if I figure it out). Should `r6OperatorCounters,json` be deleted and added to the .gitignore file if it is now generated?

Reply:
npm and node were out of date. I'm going to work on updating the README with better information. Do not worry about changing it in your PR :)

Reply:
I'm curious, was it out of date on your laptop?
My npm: v6.13.4
My node: v12.16.1

Reply:
Yep, I think my node was v10. Updated both and it worked great.